sunbird_mcp/src/server.py

```
@@ -0,0 +1,380 @@
+    """
+    try:
+        # Build API URL
+        api_url = f"{settings.API_BASE_URL.rstrip('/')}{CONTENT_ENDPOINT}/{content_id}"
+            
+        # First, get the content details with timeout
+        timeout = aiohttp.ClientTimeout(total=settings.REQUEST_TIMEOUT)
+        async with aiohttp.ClientSession(timeout=timeout) as session:
+            async with session.get(
+                api_url,
+                headers={"Content-Type": "application/json"}
+                ) as response:
```

Flake8 (7.2.0)

[error] continuation line with same indent as next logical line

(E125)